pkg/utils/mco_deploy.go

```diff
@@ -214,6 +206,10 @@ func CheckAllPodNodeSelector(opt TestOptions) error {
 			continue
 		}
 
+		if strings.Contains("grafana-test", pod.GetName()) {
```

Comment:
How does it work before? it seems you have to add `minio` as an exception as well.